src/meshapi/admin/install.py

```diff
@@ -1,13 +1,27 @@
-from typing import Tuple
+from typing import Any, Tuple
 
+import tablib
 from django import forms
 from django.contrib import admin
 from django.db.models import QuerySet
 from django.http import HttpRequest
+from import_export import resources
+from import_export.admin import ExportActionMixin, ImportExportModelAdmin
 
 from meshapi.models import Install
 
 
+class InstallImportExportResource(resources.ModelResource):
+    def before_import(self, dataset: tablib.Dataset, **kwargs: Any) -> None:
+        if "install_number" not in dataset.headers:
+            dataset.headers.append("install_number")
```

Comment:
Just out of curiosity, why this is necessary?

Reply:
IIRC this is due to our custom primary key setup. Something about the lack of auto increment